CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 # App Center SDK for React Native Change Log
 
-## Version 1.10.1 (under active development)
+## Version 1.11.0 (under active development)
 
 ### AppCenter
 
 - Introduce new `LogLevel` constants, deprecating old ones.
 
+### AppCenterPush
+
+- **[Feature]** Add manual configuration to delay the first start of Push module.
```

Comment:
I suggest something like "Support delaying Push notification permission dialog". The original feature request is at https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/287.